data/metadata.json

```diff
@@ -1,14 +1,10 @@
 {
     "clubhouse/entry/hover": {
-        "sound-file": "clubhouse/entry_hover.wav",
-        "loop": true,
-        "fade-in": 0,
```

Comment:
The zero fade times on the hover sounds were added by me originally, because if you have a nonzero fade time (for example the default for loops of one second) then every time you momentarily move the mouse over the button, you get the sound playing for one second. It's really annoying and I don't think it makes a good experience, so I think for practical considerations the fade time on this one should still be zero even if David didn't specify it.